es6-arrow-functions/es6-arrow-functions-quiz-notes.md

```diff
@@ -0,0 +1,88 @@
+# es6-arrow-functions-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is the syntax for defining an arrow function?
+- Instead of calling/Initializing a function with the function keyword you would call it with the () => with optional {}
+
+- When an arrow function's body is not surrounded in curly braces (_concise body syntax_), what changes in its functionality?
+  -Arrow funtions curly brances are indicating there is going to be more than 1 line of code inside of it.
+
+- When using _concise body syntax_, how do you return an object literal?
+  -By calling return {}
```

Comment:
Not quite! The whole point of concise body syntax is that we don't need the `return` keyword at all